packages/deploy-tooling/test/unit/base/deploy.test.ts

```diff
@@ -200,6 +200,31 @@ describe('base/deploy', () => {
             await deploy(archive, { app, target, yes: true }, nullLogger);
         });
 
+        test('Successful retry after known axios error (cloud target)', async () => {
+            const cloudTarget = {
+                cloud: true,
+                serviceKey: {
+                    uaa: {
+                        username: '~username',
+                        password: '~password',
+                        clientid: '~client',
+                        clientsecret: '~clientsecret',
+                        url: '~url'
+                    }
+                },
+                url: '~targetUrl'
+            } as AbapTarget;
+            mockedUi5RepoService.deploy.mockRejectedValueOnce(axiosError(401));
+            await deploy(archive, { app, target: cloudTarget, yes: true }, nullLogger);
+            prompts.inject(['~uaa-username', '~uaa-password']);
+            expect(mockedUi5RepoService.deploy).toBeCalledWith({
```

Comment:
Wondering if we should be more explicit what is actually changed here i.e. the serviceKey.uaa object with the updated values. Its not clear from the test?

Reply:
Good point. I've updated the test

Reply:
Thank you.